setup.py

```diff
@@ -30,23 +30,29 @@ def readme():
 except SKBuildError:
     SETUP_REQUIRES.append('cmake')
 
-setup(name='pycompwa',
-      version='0.1.00',
-      author='The ComPWA team',
-      maintainer="Peter Weidenkaff",
-      maintainer_email="weidenka@uni-mainz.de",
-      url="https://github.com/ComPWA/pycompwa",
-      description='ComPWA: The common Partial Wave Analysis framework',
-      long_description=readme(),
-      license="GPLv3 or later",
-      #  cmake_args=['-DSOME_FEATURE:BOOL=OFF'],
-      cmake_minimum_required_version=CMAKE_MINIMUM,
-      packages=find_packages(),
-      data_files=[('pycompwa/', ['./ComPWA/Physics/particle_list.xml'])],
-      zip_safe=False,
-      setup_requires=SETUP_REQUIRES,
-      tests_require=['pytest'],
-      install_requires=['wheel', 'numpy>=1.14.5', 'progress>1.3',
-                        'xmltodict>=0.11.0', 'matplotlib>=2.2.2',
-                        'uproot'],
-      )
+setup(
+    name='pycompwa',
+    version='0.1.00',
+    author='The ComPWA team',
+    maintainer="Peter Weidenkaff",
+    maintainer_email="weidenka@uni-mainz.de",
```

Comment:
This should be changed to `ComPWA Team` and `compwa-admin@ep1.rub.de`

Reply:
https://github.com/ComPWA/pycompwa/pull/74/commits/e8748a60583543d3758bd941ce4962fd17d7ce42